### ISSUE TITLE
ModuleNotFoundError when importing cupy_xarray

### ISSUE THREAD

**Issue by miguelcarcamov** [xarray-contrib/cupy-xarray#56]
I'm tryinh to import cupy_xarray as showed in the example in the README.md. I have installed cupy_xarray with `pip install cupy-xarray`. However I get this error when importing the package.

```python
---------------------------------------------------------------------------
ModuleNotFoundError                       Traceback (most recent call last)
Cell In[10], line 1
----> 1 import cupy_xarray

File ~/.conda/envs/pyralysis-env/lib/python3.12/site-packages/cupy_xarray/__init__.py:2
      1 from . import _version
----> 2 from .accessors import CupyDataArrayAccessor, CupyDatasetAccessor  # noqa
      4 __version__ = _version.get_versions()[\"version\"]

File ~/.conda/envs/pyralysis-env/lib/python3.12/site-packages/cupy_xarray/accessors.py:8
      1 import cupy as cp
      2 from xarray import (
      3     DataArray,
      4     Dataset,
      5     register_dataarray_accessor,
      6     register_dataset_accessor,
      7 )
----> 8 from xarray.core.pycompat import DuckArrayModule
     10 dsk = DuckArrayModule(\"dask\")
     11 dask_array_type = dsk.type

ModuleNotFoundError: No module named 'xarray.core.pycompat'"
```

Versions:
```python
cupy-xarray==0.1.3
xarray==2024.6.0
```

**Comment by jacobtomlinson** [comment]
Hmm it looks like that submodule has been removed. We will need to fix that. In the meantime please use `xarray==2024.01.1`.

**Comment by keewis** [comment]
see also #46 and #50

**Comment by jacobtomlinson** [comment]
Nice! Let's close this as a duplicate of #50 then.

**Comment by weiji14** [comment]
Yes, I'm happy to do a patch release with the bugfix. Just waiting on getting permissions to set up versioning on the readthedocs page - https://github.com/xarray-contrib/cupy-xarray/pull/51#issuecomment-2197765656
